tests/test_fastapi.sh

```diff
@@ -7,6 +7,8 @@ cd fastapi
 git fetch --tags
 git checkout 0.99.1
 
+# temp fix for flask dependency issue
+pip install Werkzeug==2.2.2
```

Comment:
What's the issue?

Reply:
@Kludex https://stackoverflow.com/questions/77213053/importerror-cannot-import-name-url-quote-from-werkzeug-urls

Reply:
can you add a link in the comment to this pr, to help anyone else finding this.